internals/scripts/init-github-project.js

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
+
+require('shelljs/global');
+const inquirer = require('inquirer');
+const fs = require('fs');
+
+inquirer.prompt([{
+    type: 'input',
+    name: 'name',
+    message: 'What is your project name?',
+    default: 'my-boilerplate-project',
+    validate: (value) => /.+/.test(value) ? true : 'The name is required'
+  }, {
+    type: 'input',
+    name: 'description',
+    message: 'What is your project description?',
+    default: 'a project created with react-boilerplate',
+    validate: (value) => /.+/.test(value) ? true : 'The description is required'
+  }, {
+    type: 'input',
+    name: 'author',
+    message: 'Who is your the project author?',
```

Comment:
A little nitpick with the wording here: 'Who is your the project author?' could be 'Who is the project author?' or 'What is the name of the project author?'